webapps/frontend/webpack.common.js

```diff
@@ -170,6 +165,22 @@ module.exports = {
         {
           from: path.resolve(__dirname, 'public'),
           to: './'
+        },
+        {
+          from: path.resolve(
+            __dirname,
+            'node_modules/requirejs-angular-define/dist/ngDefine.js'
+          ),
+          to: './lib',
+          noErrorOnMissing: true
+        },
+        {
```

Comment:
❓The second resolve is for EE?

Reply:
Yes. Ugly, but I found no other way to do it.